packages/jupyter/src/providers.tsx

```diff
@@ -105,7 +105,7 @@ export function notebookFromMdast(
       const cell = new core.ThebeNonExecutableCell(
         block.key,
         notebook.id,
-        block.children.reduce((acc, child) => acc + '\n' + (child.value ?? ''), ''),
+        block.children?.reduce((acc, child) => acc + '\n' + (child.value ?? ''), ''),
```

Comment:
I think the if-statement will also fall through to here. This will mean that the content is undefined?

Is that fine? I am not sure what the `ThebeNonExecutableCell` does in this case!

Reply:
nice catch -- the no children condition only occurs with empty markdown cells, so we want to create the non executable cell with an empty source string in that case